wcfsetup/install/files/style/icon/vendor/_variables.scss

```diff
@@ -0,0 +1,1368 @@
+// Variables
+// --------------------------
+
+$fa-font-path:                "../font/" !default;
```

Comment:
Check whether it is easy to support `getFont.php` using this variable (it might require a change to `getFont.php`).